hadoop-master1/files/etc/trino/conf/config.properties

```diff
@@ -0,0 +1,4 @@
+coordinator=true
+node-scheduler.include-coordinator=true
```

Comment:
typically should be `false` in production deployment?

Reply:
Yes, currently single-node multiple roles that only for testing.